src/Plugins/StorageDumper/StorageDumper.cs

```diff
@@ -176,7 +176,7 @@ private string GetOrCreateDirectory(uint network, uint blockIndex)
 
         private string GetDirectoryPath(uint network, uint blockIndex)
         {
-            uint folder = (blockIndex / Settings.Default.StoragePerFolder) * Settings.Default.StoragePerFolder;
+            uint folder = (blockIndex / Settings.Default!.StoragePerFolder) * Settings.Default.StoragePerFolder;
```

Comment:
`Setttings.Default` is not nullable, isn't it?

Reply:
Yes it is

Reply:
Yes, its `nullable`.

Reply:
> `Setttings.Default` is not nullable, isn't it?

The default is very weird, cause its initialized when the load is called, no real default value for it.